ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/forkchoice/ReadOnlyStore.java

```diff
@@ -48,4 +52,8 @@
   BeaconState getCheckpointState(Checkpoint checkpoint);
 
   boolean containsCheckpointState(Checkpoint checkpoint);
+
+  VoteTracker getVote(int validatorIndex);
+
+  Collection<VoteTracker> getVotes();
```

Comment:
I would only add these to `MutableStore`.  Seems weird but `VoteTracker` is mutable so you can't be allowed to get one from an immutable store.

Reply:
Probably don't actually need `getVotes` either - just `getVote` and maybe a `getVoteCount` but not sure even that will be required.

Reply:
Makes sense.